### PR TITLE
Fix a5 config and update custom files

### PR DESCRIPTION
**Pull request by eliasbakken**

### CODE REVIEW

File: userpatches/overlay/klipper/generic-recore-a5.cfg [hunk 1]
```diff
@@ -248,11 +257,17 @@ pin: PB5
 pin: PB4
 
 # Set up board voltage, current, temperature.
```
Comment (IfOnlyItWasThatEasy) [inline userpatches/overlay/klipper/generic-recore-a5.cfg:259]: This line (the new line 259) is a duplicate

**Review by IfOnlyItWasThatEasy — COMMENTED** [review]
Thanks for making the changes! I tried the files as-is, and the results are much better. I can now turn on the heaters, and I have temperature readings for thermistors. The thermistor readings are still 8-9 degrees C low, but it's much better!

File: userpatches/overlay/klipper/generic-recore-a5.cfg
Comment (IfOnlyItWasThatEasy) [inline userpatches/overlay/klipper/generic-recore-a5.cfg:1]: I checked this config on my A5 board and didn't receive any errors. Steppers buzzed, thermistors provide output, and heater outputs activate.

Comment (IfOnlyItWasThatEasy) [inline userpatches/overlay/klipper/generic-recore-a5.cfg:1]: I just realized that removing the gcode_ids in lines 309, 327, and 341 break the outputs at the top of the Octoprint window. I recommend leaving them in. I'm not sure where the gcode_id in line 317 is used at.

**Comment by eliasbakken** [comment]
Yeah, I think I've noticed the same thing. I was told by the klipper guys
that is was not necessary, but perhaps that only applies to Fluidd and
MainSail then. I'll add it back. Thanks!

tir. 4. jun. 2024, 00:58 skrev IfOnlyItWasThatEasy ***@***.***

**Pull request merged**